File: Top 100 Codes/01-Getting Started/01-check if a Number Is Positive Or Negative/checkifaNumberIsPositiveOrNegative.py
# Prompt the user to input a number
num = int(input("Enter a number: "))  # Convert the input to an integer

# Check if the number is greater than or equal to zero
if num >= 0:
    # If the number is equal to zero
    if num == 0:
        print('Zero')  # Print "Zero" for the number 0
    else:
        print("Positive")  # Print "Positive" for numbers greater than 0
else:
    print("Negative")  # Print "Negative" for numbers less than 0

File: Top 100 Codes/01-Getting Started/01-check if a Number Is Positive Or Negative/checkifaNumberIsPositiveOrNegative.cpp
#include <iostream> // Include the input-output stream library
using namespace std;

int main() {
    int num; // Variable to store the user input

    // Prompt the user to enter a number
    cout << "Enter a number: ";
    cin >> num; // Take user input

    // Condition to check if the number is negative or positive
    if (num >= 0) {
        if (num == 0) {
            cout << "Zero"; // Output if the number is 0
        } else {
            cout << "The number is positive"; // Output if the number is positive
        }
    } else {
        cout << "The number is negative"; // Output if the number is negative
    }

    return 0; // Indicate successful execution
}
